core/services/ocr2/plugins/ocr2keeper/evm/registry.go

```diff
@@ -586,9 +586,9 @@ func (r *EvmRegistry) checkUpkeeps(ctx context.Context, keys []types.UpkeepKey)
 // TODO (AUTO-2013): Have better error handling to not return nil results in case of partial errors
 func (r *EvmRegistry) simulatePerformUpkeeps(ctx context.Context, checkResults []types.UpkeepResult) ([]types.UpkeepResult, error) {
 	var (
-		performReqs     = make([]rpc.BatchElem, 0, len(checkResults))
-		performResults  = make([]*string, 0, len(checkResults))
-		performToKeyIdx = make([]int, 0, len(checkResults))
+		performReqs     []rpc.BatchElem
+		performResults  []*string
+		performToKeyIdx []int
```

Comment:
Leave the capacity definitions in place since we already know the max array size. Append will still work the same way, but the go runtime won't have to guess on array sizing.